src/unix/linux_like/linux/gnu/b32/arm/mod.rs

```diff
@@ -61,7 +61,10 @@ s! {
     pub struct stat64 {
         pub st_dev: crate::dev_t,
         __pad1: c_uint,
-        __st_ino: crate::ino_t,
+        // #[cfg(not(gnu_file_offset_bits64))]
+        // __st_ino: crate::ino_t,
+        // #[cfg(gnu_file_offset_bits64)]
+        __st_ino: c_ulong,
```

Comment:
What is the reason for this to be commented out?

Reply:
This was included by mistake. I'll remove it.